travis/travis_install_nightly

```diff
@@ -84,7 +84,17 @@ fi;
 
 if [ "${WKHTMLTOPDF_VERSION}" != "" ]; then
     echo "Install webkit (wkhtmltopdf) patched version ${WKHTMLTOPDF_VERSION}"
-    (cd ${HOME}/maintainer-quality-tools/travis/ && wget -qO- -t 1 --timeout=240 https://downloads.wkhtmltopdf.org/0.12/${WKHTMLTOPDF_VERSION}/wkhtmltox-${WKHTMLTOPDF_VERSION}_linux-generic-amd64.tar.xz | tar -xJ --strip-components=2 wkhtmltox/bin/wkhtmltopdf)
+
+    cd ${HOME}/maintainer-quality-tools/travis/ &&
+    wkhtmltox=$(wget -qO- -t 1 --timeout=240 https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox-${WKHTMLTOPDF_VERSION}_linux-generic-amd64.tar.xz) &&
+    WKHTMLTOX_CHECKSUM='049b2cdec9a8254f0ef8ac273afaf54f7e25459a273e27189591edc7d7cf29db'
```

Comment:
This isn't being exported though, so it doesn't survive past the execution of this script - or am I wrong?

The alternative is just checking the sum against a string, which I do not feel is as clear.

> I mean, we could use a wget_custom with a dictionary of checksum and url used from MQT.

IMO this is another feature in itself. We currently support one version of wkhtmltopdf according to what I see here.